packages/components/src/Lifecycle/index.js

```diff
@@ -6,6 +6,10 @@ class Lifecycle extends React.Component {
     this.props.onMount && this.props.onMount();
   }
 
+  componentDidUpdate() {
+    this.props.onUpdate && this.props.onUpdate();  
```

Comment:
We should pass through all the arguments from didUpdate so you can compare props if necessary